test/test.repo.js

```diff
@@ -272,14 +273,35 @@ describe('Creating new Github.Repository', function() {
    it('should get pull requests on repo', function(done) {
       var repo = github.getRepo('michael', 'github');
 
-      repo.getPull(153, function(err) {
+      repo.getPull(153, function(err, pull) {
```

Comment:
Can you remove this change and move it into another PR? This is not related to the merge feature.